src/App.css

```diff
@@ -54,4 +26,25 @@
 .article_image {
 width: 250px;
 max-height: 250px;
+}
+.individual_article_card {
```

Comment:
This element is a bit off center and strange looking on the page - try playing around with the settings a bit. When I edited width to auto and got rid of the padding, it improved the visuals a lot.